specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/networkSecurityGroup.json

```diff
@@ -870,6 +870,19 @@
     },
     "NetworkSecurityGroupPropertiesFormat": {
       "properties": {
+        "flushConnection": {
+          "type": "string",
+          "description": "When enabled, flows created from Network Security Group connections will be re-evaluated when rules are updates. Initial enablement will trigger re-evaluation.",
+          "enum": [
+            "Enabled",
+            "Disabled",
+            "null"
```

Comment:
could you put null at the top of enum values?  That would be helpful for SDK generation that 'null' is the first item of the enum.